.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,240 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: R
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.config.os-name }}-${{ matrix.config.os-version }}
+    name: >- 
+      ${{ matrix.config.os-name }}-${{ matrix.config.os-version }}
+      R ${{ matrix.config.r-version}} - java ${{ matrix.config.java}}
+    strategy:
+      matrix:
+        config: 
+          - os-name: ubuntu # windows-2019, macos-10.15]
+            os-version: latest
+            r-version: oldrel
+            java: 15
+            vignettes: false
+            xlc-repo: https://jcenter.bintray.com
+          - os-name: ubuntu # windows-2019, macos-10.15]
+            os-version: "18.04"
+            r-version: release
+            java: 11
+            vignettes: true
```

Comment:
We seem to be using `vignettes` as a way to do a full check `--as-cran` (covering vignettes as well).

According to CRAN policies, we should guarantee a full check `--as-cran` also on `oldrel` and `develop`.

Reply:
Added `vignettes=true` to each R of these ubuntu R builds